styles/style.css

```diff
@@ -6,3 +6,128 @@ Green: #00B172
 White: #FFF
 
 */
+body{
+    margin:0;
+    padding: 0;
+}
+
+.nav {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    height: 50px;
+    position: fixed;
+    background-color: white;;
+    justify-content: space-between;
+}
+
+.nav img {
+    height: 40px;
+}
+
+.nav-bar{
+    justify-content: center;
+}
+
+.nav-bar ul{
+    margin: 0;
+    width: 450px;
+    display: flex;
+    justify-content: space-around;
+    list-style-type: none;
+    align-self: center;
+}
+
+.nav a {
+    text-decoration: none;
+    color:black;
+}
+
+#header{
+    height: 650px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    background-image: url("../images/landing.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: bottom;
+    color: white;
+}
+
+h1 {
+    font-size: 5em;
+    margin: 0;
+}
+
+p{
+    font-size: 18px;
+}
+
+#general-information{
+    height: 400px;
+    text-align: center;
+    margin-top: 50px;
+}
+
+#general-information hr {
+    border: 1px solid #00B172;
+    width: 200px;
+}
+
+.general-information-container {
```

Comment:
Is there a reason why you use a class here rather than an id? If not, as you are not using this styling anywhere else, I would use an id. 
And could you nest this in the #general-information id somehow?